src/styles/image.css

```
@@ -0,0 +1,10 @@
+.card__img-wrapper {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
```

```suggestion
  margin: 0 auto 40px;
```

Чтоб блок был независимым у него не должно быть внешних отступов так как после его переноса в другое место его внешние отступы могу мешать рядом стоящим элементам